README.md

```diff
@@ -174,4 +174,9 @@ struct SubCommandTwo {
 }
 ```
 
+It is also possible to retrieve just the names of the arguments and switches 
+passed at runtime, by including the switch `--dump_args_passed` in the args, similarly
```

Comment:
name updated to match style in latest commit.

> Could you also make sure that this flag shows up in `--help`?

Should this be included in --help output or would it confuse an end user of a tool built using argh since it mirrors back to them the options and subcommands they entered?